openedx/core/djangoapps/programs/models.py

```diff
@@ -132,3 +137,10 @@ def is_certification_enabled(self):
         certificates for Program completion.
         """
         return self.enabled and self.enable_certification
+
+    @property
+    def show_xseries_ad(self):
```

Comment:
I understand the concern here, but at this moment in time, not all programs are created equal. I think we don't yet fully understand the strategy of the program types at the moment to make these advertising more generic. I say let's leave this as it is to explicitly call out we show xseries advertising for now. It's makes future improvement easier when we are ready to make these more generic.